v1/rest/rest.go

```diff
@@ -186,3 +190,57 @@ func GetData(jsonBody []byte) (json.RawMessage, error) {
 	data := body["body"]
 	return data, err
 }
+
+// PublishLog params
+// @context: *gin.Context
+// status: int
+// payload: interface
+// msg: string
```

Comment:
```suggestion
// PublishLog params
//  @context: *gin.Context
//  @status: int
//  @payload: interface
//  @msg: string
```
sebelum `@` tambahin tab sekali pak